README.md

```diff
@@ -12,7 +12,7 @@ useful for users new to the plugin and users new to Atom as well.
 
 ## What is it?
 
-This package is a full-featured Dart development plugin for Atom. It supports
+This package is a Dart development plugin for Atom. It supports
```

Comment:
lgtm but we should follow up w/ some other adjectives to punch this sentence up. Perhaps lightweight? streamlined?